lang/en/admin/apikey.php

```diff
@@ -0,0 +1,25 @@
+<?php
+
+return [
+    'title' => 'Application API Keys',
+    'emptytable' => 'No API keys.',
```

Comment:
Quick find an replace!

```suggestion
    'empty_table' => 'No API keys.',
```